marlowe-integration-tests/marlowe

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+cabal run marlowe-runtime-cli -- $@
```

Comment:
Let's remove this and create a script using nix that can be added to the dev shell that does this.